src/overlays/actors/ovl_En_Bsb/z_en_bsb.h

```diff
@@ -11,7 +11,9 @@ typedef struct EnBsb {
     /* 0x0000 */ Actor actor;
     /* 0x0144 */ char unk_144[0x140];
     /* 0x0284 */ EnBsbActionFunc actionFunc;
-    /* 0x0288 */ char unk_288[0xED0];
+    /* 0x0288 */ char unk_288[0x54];
+    /* 0x02DC */ s32 unk2DC;
```

Comment:
This is limbCount in EnSkb, worth assuming that it's the same here?

Reply:
Not really, probably best to leave as unk for now.